src/Controller/ContactController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ContactController extends AbstractController
+{
+    #[Route('/contact', name: 'app_contact')]
```

Comment:
This is a good approach for using annotations and I appreciate, but for our project we will use the yaml config files.
comment this line and use config/routes.yaml configuration